spec/rectangle_spec.rb

```diff
@@ -1,7 +1,17 @@
 require_relative '../lib/screenshot.rb'
 
 # Rectangle is defined as set of co-ordinates represented by top left x, top left y, width, height
+# Iframe Offset rectangle will always have width and height to be 0, because these parameters
```

Comment:
Yup ! after this patch it is no longer about just iframes, but also padding. You can say it like this: 

Offset can be used either by providing direct rectangle co-ordinates or by iframe element (but here the good part is, if you provide iframe, then elements inside the iframe will be cropped[This was initial functionality which is still maintained])